commands/kubernetes.go

```diff
@@ -547,6 +548,27 @@ func RunKubernetesOneClickList(c *CmdConfig) error {
 	return c.Display(items)
 }
 
+// RunKubernetesOneClickInstall installs 1-click apps on a kubernetes cluster.
+func RunKubernetesOneClickInstall(c *CmdConfig) error {
+	oneClicks := c.OneClicks()
+	if len(c.Args) < 1 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	oneClickSlice, err := c.Doit.GetStringSlice(c.NS, doctl.ArgOneClicks)
+	if err != nil {
+		return err
+	}
+
+	OneClickInstall, err := oneClicks.InstallKubernetes(c.Args[0], oneClickSlice)
```

Comment:
Having an uppercase letter for a locally scoped variable (instead of a global, exported one) is a bit confusing, so it should start with a lowercase letter.
```suggestion
	oneClickInstall, err := oneClicks.InstallKubernetes(c.Args[0], oneClickSlice)
```